workers/unity/Assets/Playground/Scripts/MonoBehaviours/ToggleRotationCommandSender.cs

```diff
@@ -0,0 +1,37 @@
+using Generated.Playground;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+using UnityEngine;
+
+namespace Playground.MonoBehaviours
+{
+    public class ToggleRotationCommandSender : MonoBehaviour
+    {
+        [Require] private SpinnerRotation.Requirables.Reader reader;
+        [Require] private SpinnerRotation.Requirables.CommandRequestSender requestSender;
+        private EntityId ownEntityId;
+
+        private void OnEnable()
+        {
+            if (reader != null) // TODO UTY-791: Needed until prefab preprocessing is implemented, remove as part of UTY-791
+            {
+                ownEntityId = GetComponent<SpatialOSComponent>().SpatialEntityId;
+            }
+        }
+
+        private void Update()
+        {
+            if (reader.Authority != Authority.NotAuthoritative)
```

Comment:
At this point it's worth considering if we want to have separate prefabs for server vs client side, which is something we could do with the existing PrefabConfig... I personally think this is OK for now to keep it simple for quick tests though.